_examples/simple.go

```diff
@@ -29,7 +29,7 @@ func main() {
 }
 
 func produce(client *sarama.Client, n int) error {
```

Comment:
`sarama.Client` is now an interface in the latest sarama version, please change to `func produce(client sarama.Client, n int) error`